src/config.ts

```diff
@@ -20,7 +19,7 @@ const chains: Chain[] = [
   {
     id: 1,
     name: "mainnet",
-    rpc: `https://mainnet.infura.io/v3/${process.env.INFURA_API_KEY}`,
+    rpc: `https://mainnet.infura.io/v3/${process.env.INFURA_API_KEY ?? ""}`,
```

Comment:
Not for this PR but I think that eventually we should have a `getChains` method that returns this object and that throws when the necessary API KEYs aren't provided, so we don't start the service with bad configuration.

Reply:
@bard that's a great idea, just a note though is that both the Infure and Alchemy API keys are not required, the indexer should still work, might get some more rate limit errors though, but could be worth making them required.